course-v2/layouts/partials/partial_collapse_list.html

```diff
@@ -1,5 +1,5 @@
 {{ $params := . }}
-{{ $className := .klass | default "coming-soon" }}
+{{ $className := .klass | default "" }}
```

Comment:
All instances of `partial_collapse_list` in course-v2 were passing an explicit `klass` parameter, so the default was never getting used.